clang/lib/Lex/Lexer.cpp

```diff
@@ -2270,9 +2270,11 @@ bool Lexer::LexRawStringLiteral(Token &Result, const char *CurPtr,
       const char *PrefixEnd = &CurPtr[PrefixLen];
       if (PrefixLen == 16) {
         Diag(PrefixEnd, diag::err_raw_delim_too_long);
-      } else {
+      } else if (*PrefixEnd != '\n') {
```

Comment:
I implemented it in that way initially, but the PrefixLen variable is assigned with the last token of the translation file (which is usually \n) when it could not find '(' token. So, I couldn't find the practical scenario where the *PrefixEnd will be other than \n. Hence I flipped the cases.
Can you suggest any scenarios if you can remember on top of your head?

Reply:
hmmm. what happens if the line ends with `\r\n`?

Reply:
ummm... still the file will end with \n right?

Reply:
couple of examples: https://clang.godbolt.org/z/161bn5d6s

Reply:
Great! Thanks, I will flip the conditions. and update the testcases.